frame/clearing-house/Cargo.toml

```diff
@@ -30,6 +30,13 @@ scale-info = { version = "1.0", default-features = false, features = [
 ] }
 serde = { version = '1', optional = true }
 
+[dev-dependencies]
+orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", rev = "2b1c9fb367ccb8e13601b2da43d1c5d9737b93c6", default-features = true }
```

Comment:
I turned `default-features = true` here since without it I was running into the following error via `cargo check -p clearing-house --all-targets`
```zsh
error: `Tokens` does not have the std feature enabled, this will cause the `orml_tokens::GenesisConfig` type to be undefined.
  --> frame/clearing-house/src/mock/runtime.rs:22:1
   |
22 | / frame_support::construct_runtime!(
23 | |     pub enum Runtime where
24 | |         Block = Block,
25 | |         NodeBlock = Block,
...  |
34 | |     }
35 | | );
   | |_^
   |
   = note: this error originates in the macro `orml_tokens::__substrate_genesis_config_check::is_std_enabled_for_genesis` (in Nightly builds, run with -Z macro-backtrace for more info)
```
Not sure this is the best solution though, as I've seem other pallets use the `orml_tokens` pallet in their mocks without turning this on (e.g., the mocks for `frame/dutch-auction`)

Reply:
add std and runtime-benchamarks for each pallet you add and default features false